src/data/linux-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const linuxQuiz = [
+  {
+    Question: "In Linux, which command grants every permission to everyone?",
+    Answer: "chmod 777",
+    Distractor1: "chmod 111",
+    Distractor2: "chmod 724",
+    Distractor3: "chmod 666",
+    Explanation:
+      "The command chmod 777 activates all the permission access to owner, group and others by summing 4, 2, 1 on octal base.",
```

Comment:
small change, but it will read better this way. 

```suggestion
      "The command chmod 777 activates all the permission access to owner, group and others by summing 4, 2, 1 on the octal base.",
```